src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,39 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs,
+  selectedTabId,
+  setSelectedTabId,
+  onTabSelected }) => {
+  const click = (tabId, selectedId, tabsList) => {
```

Comment:
we usually name it like this
```suggestion
  const handleClick = (tabId, selectedId, tabsList) => {
```